vrs/marten-construction-script.py

```diff
@@ -1,168 +1,180 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# Daniel Marten 01-04-2022
-# Constructing a ~100k Variant Test Database for GA4GH-VRS Annotation Script
-# NOTE: adapted from a Jupyter Notebook, so excuse any formatting quirks! 
-# NOTE: please make sure to change the paths for checkpointing!!!
-# Edits on: 01-06-2022
+# Daniel Marten 01-20-2023
+# Construct a ~100k Variant Test VCF for GA4GH-VRS Project
+# Reach out to me at: marten@broadinstitute.org
 
+# Import and set up important packages/modules
 import hail as hl
 import random
 from random import sample
 from gnomad.resources.grch38.gnomad import public_release
 
-# Setting randomness
+from gnomad.resources.resource_utils import (
+    MatrixTableResource,
+    VariantDatasetResource,
+    VersionedMatrixTableResource,
+    VersionedVariantDatasetResource,
+)
+
+# Import logging for logs (to replace print statements)
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("create_vrs_test_dataset")
+logger.setLevel(logging.INFO)
+
+# Set randomness for python package random
 random.seed(505)
```

Comment:
this could be something like:
```
random.seed(args.random_seed)
```
to parameterize this number (rather than hardcoding), and the init below could be:
```
hl.init(default_reference="GRCh38", global_seed=args.hail_seed, tmp_dir="gs://gnomad-tmp-4day/vrs")
```
^I missed this before, but you'll probably want to set the default reference to build 38 to be safe (the default is build 37; see https://hail.is/docs/0.2/api.html#hail.init). as another hail tip, adding a tmp_dir to hail inits will help optimize scripts (hail will put temp files in that bucket rather than depending on hdfs space on your cluster)